src/ng-xform/ng-xform.component.spec.ts

```diff
@@ -92,10 +91,6 @@ describe('NgXformComponent', () => {
     expectFormTextarea('comments', 'Comments', 'comments here...');
   });
 
-  it('should render MeasureField', () => {
```

Comment:
What should be rendered on measure field declaration?

Reply:
This test has been moved to input-number.component.spec.ts. This specification is the first created and has tests of all the components, but we want to test each component separately. Gradually we move and we will get there! :wink: